releases/2.13.11.md

```diff
@@ -51,6 +51,15 @@ The following changes are highlights of this release:
 
 For the complete 2.13.11 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.11) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.11).
 
+## Known issues
+
+A few regressions have already cropped up:
+
+* Exhaustivity checker emits spurious warning when matching on Java enum type (2.13.11 regression) ([scala/bug#12800](https://github.com/scala/bug/issues/12800))
+* Duplicated `@Deprecated` annotations when implementing deprecated Java interface cause `java.lang.annotation.AnnotationFormatError` when accessed via Java reflection (2.13.11 regression) ([scala/bug#12799](https://github.com/scala/bug/issues/12799))
```

Comment:
`when extending a Java interface with a deprecated default method`. As my Grammie used to say, Cut 'em thin, sure to win.

Reply:
thanks. the lengthy/detailed description helps convey the obscurity of the corner case